### PR TITLE
Ensure the daemonset pods all have cpu/mem request set.

### PR DESCRIPTION
**Pull request by awels**
Signed-off-by: Alexander Wels <awels@redhat.com>

**What this PR does / why we need it**:
Request some minimum cpu/mem so we don't get evicted first due to not having them set.

**Which issue(s) this PR fixes** *(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close the issue(s) when PR gets merged)*:
Fixes #

**Special notes for your reviewer**:

**Release note**:

```release-note
NONE
```

### CODE REVIEW

File: pkg/controller/hostpathprovisioner/daemonset.go [hunk 1]
```diff
@@ -584,7 +604,13 @@ func (r *ReconcileHostPathProvisioner) createCSIDaemonSetObject(cr *hostpathprov
 							TerminationMessagePolicy: corev1.TerminationMessageReadFile,
 						},
 						{
-							Name:            nodeDriverRegistrarName,
```
Comment (akalenyu) [inline pkg/controller/hostpathprovisioner/daemonset.go:587]: The const is still there though? not sure why?

Comment (awels) [reply pkg/controller/hostpathprovisioner/daemonset.go:587]: rebase failure on my part

**Pull request merged**